karapace/config.py

```diff
@@ -39,9 +39,11 @@
     "port": 8081,
     "server_tls_certfile": None,
     "server_tls_keyfile": None,
+    "registry_url": None,
```

Comment:
README doesn't mention `registry_url`. What's the difference that and `registry_host`?

Reply:
My bad, did originally `registry_url` containing user & pw, but changed it after discussion to current form. This was leftover from first iteration.